db/fixtures/seeds.rb

```diff
@@ -1,6 +1,6 @@
 
 Conference.seed(
-  {id: 1, name: "CloudNative Days Tokyo 2020", abbr: "cndt2020", status: 0}
+  {id: 1, name: "CloudNative Days Tokyo 2020", abbr: "cndt2020", status: 2}
```

Comment:
```suggestion
  {id: 1, name: "CloudNative Days Tokyo 2020", abbr: "cndt2020"}
```

seedから消したらDB側のデータが変わらないのでそっちがいいかな